sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -47,9 +47,9 @@ public class TimeSeriesInsightsClient
         public virtual TimeSeriesInsightsHierarchies Hierarchies { get; private set; }
 
         /// <summary>
-        /// Query client that can be used to perform query operations on Time Series Insights.
+        /// Queries client that can be used to perform query operations on Time Series Insights.
```

Comment:
Still hard for me to get used to calling it Queries client and not Query client

Reply:
It's a good point to bring up during today's API review

Reply:
Perhaps a bit of rearranging? Does this make sense given the nature of the property/class?

```suggestion
        /// A client that can be used can be used to perform query operations on Time Series Insights.
```